app/controllers/political_controller.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+class PoliticalController < ApplicationController
+  before_action :check_pre_release_permission
+  before_action :check_permission
+
+  def edit
+    @edition = Edition.find_current(document: params[:document])
+    assert_edition_state(@edition, &:editable?)
+  end
+
+  def update
+    result = Political::UpdateInteractor.call(params: params, user: current_user)
+    redirect_to document_path(result.edition.document)
+  end
+
+  def check_pre_release_permission
+    return if current_user.has_permission?(User::PRE_RELEASE_FEATURES_PERMISSION)
+
+    head :forbidden
```

Comment:
I think rather than needing to do a conditional approach like this we can call have gds-sso handle it - check https://github.com/alphagov/content-publisher/blob/d95207d9118738e11a01382bd4e29da28bae3a01/app/controllers/debug_controller.rb#L4

That'll also mean a user gets served a page explaining too.